test-context/src/lib.rs

```diff
@@ -24,9 +25,13 @@ pub struct TrustifyContext {
     pub graph: Graph,
     pub storage: FileSystemBackend,
     pub ingestor: IngestorService,
+    pub mem_limit_mb: f32,
     postgresql: Option<PostgreSQL>,
 }
 
+#[global_allocator]
```

Comment:
question: this doesn't find its way into the final binary, yeah? Purely when testing?

Reply:
Confirmed :+1: 
https://github.com/helio-frota/mango?tab=readme-ov-file#mango